sql/program_attribute_types.sql

```diff
@@ -0,0 +1,10 @@
+-- update program names: This is done because the initializer sets the name as the concept name but we want to use a key instead
```

Comment:
@rubailly could you add this file a `configuration/scripts/` path instead?

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/12/commits/bcae562d21274d5d66d49fc8b4c3a27778037ee6?file-filters%5B%5D=)